raiden_libs/messages/__init__.py

```diff
@@ -2,10 +2,12 @@
 from .balance_proof import BalanceProof
 from .monitor_request import MonitorRequest
 from .fee_info import FeeInfo
+from .request_paths import RequestPaths
 
 __all__ = [
     'Message',
     'BalanceProof',
     'MonitorRequest',
-    'FeeInfo'
+    'FeeInfo',
+    'RequestPaths'
```

Comment:
We should specify some naming conventions here. There is the `MonitorRequest` which implies that it should be named `PathsRequest`.